rules-reviewed/jboss-eap/websphere/websphere.windup.xml

```diff
@@ -0,0 +1,701 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/v1/xml" id="websphere_rules_general">
+    <rules>
+        <rule id="websphere_rules_general-7Apvvd">
```

Comment:
I don't think that it was discussed, but I do think that it would be a good convention to follow.